src/ArrayLayouts.jl

```diff
@@ -148,7 +156,7 @@ end
 macro layoutgetindex(Typ)
     esc(quote
         ArrayLayouts.@_layoutgetindex $Typ
-        ArrayLayouts.@_layoutgetindex LinearAlgebra.AbstractTriangular{<:Any,<:$Typ}
+        ArrayLayouts.@_layoutgetindex UpperOrLowerTriangular{<:Any,<:$Typ}
```

Comment:
Needs `ArrayLayouts.UpperOrLowerTriangular` in case this macro is called from outside the package.